services/hashtagService.js

```diff
@@ -5,3 +5,9 @@ export default async function getHashTags() {
 
   return hashTags;
 }
+
+export default async function addHashTags(tag) {
+  const hashTag = await HashTag.create({ tag });
+
+  return hashTags;
+}
```

Comment:
@mango906 서비스 함수 구조 무척 깔끔해 보입니다.

혹시 `export default` 선언을 여러 번 할 수 없지 않을까요~?
다음과 같이 default 를 제거하여야
`routes/posts/hashTags/index.js` L:2 에서 import 구조 분해가 이루어지지 않을까 합니다~

```suggestion
export async function getHashTags(...) {
  ...
}

export async function addHashTags(tag) {
  const hashTag = await HashTag.create({ tag });

  return hashTags;
}
```

Reply:
@kor-Chipmunk 그런거 같네요.. 수정해놓도록 하겠습니다  default의 역할에 대해서는 알고 있었는데 한 파일에서 한번만 쓸 수 있다는건 처음 알았네요 감사합니다!
(default가 파일 전체를 export한다는 개념으로 보면 편할거 같더라구여)

https://medium.com/@_diana_lee/default-export%EC%99%80-named-export-%EC%B0%A8%EC%9D%B4%EC%A0%90-38fa5d7f57d4